src/vue-horizontal-list.vue

```diff
@@ -75,7 +75,16 @@
       options: {
         type: Object,
         required: false
-      }
+      },
+      /**
+      * From the parent component you can send propPosition to set the initial slide. 
+      * Also since prop values are passed down to child components you can now control the slides from parent component
+      * For e.g. You have a list with 5 items. You can now start the horizontal slides from the 2nd item
+      */
+      propPosition: {
+        type: Number,
+        required: false
+      },
```

Comment:
Thanks, I will merge this. But I will change how it's implemented. Targeting this for v1.2.0.

Reply:
good improvement, this is sure welcome feature, @fuxingloh shouldn't be the startOnIndex in autoplay be deprecated by this new one implementation?

Reply:
I see you have fixed this in https://github.com/fuxingloh/vue-horizontal-list/pull/41/commits/8a5cd687b1c13f2c710f22e0c05642e2f18f897d